source/disruption/traffic_reports_api.cpp

```diff
@@ -343,8 +343,17 @@ void TrafficReport::disruptions_list(const std::string& filter,
         return;
     }
 
-    type::Indexes network_idx = ptref::make_query(type::Type_e::Network, filter, forbidden_uris, d);
-    add_networks(network_idx, d, now, filter_period);
+    type::Indexes network_idx;
+    try {
+        network_idx = ptref::make_query(type::Type_e::Network, filter, forbidden_uris, d);
+    } catch (const std::exception&) {
+    }
```

Comment:
a log may be helpful

Reply:
Here i don't see any use of log as we want to continue even if no network exists for the filter.